src/plot-recipes.jl

```diff
@@ -17,3 +17,60 @@ end
     color        := :grays
     img.data[1]
 end
+
+@recipe function f(img::AstroImage, wcs::WCSTransform)
```

Comment:
I think that this recipe applies only to a single colour image (`AstroImg{T,C,1}`), what do you think?

Reply:
Yeah it's only for a single colour now. The coloured ones are a simple tweak and is waiting for #15.
I explicitly wrote it this way because before #15 gets in, test won't pass otherwise.
I'm looking into #15 to solve the CI errors.